docs/content/introduction/begrippenlijst.md

```diff
@@ -0,0 +1,29 @@
+## Begrippenlijst
+In onderstaande begrippenlijst worden begrippen zoals deze gebruikt worden in de verschillende content van deze repository toegelicht. 
+
+- BSM: Bericht Structuur Model, een UML klassediagram dat  is afgeleid van het UGM waarmee de structuur van een specifiek bericht wordt vastgelegd.
+- DRC: Documentregistratiecomponent. Component voor opslag en ontsluiting van documenten en daarbij behorende metadata. De component ondersteunt het opslaan en naar andere applicaties ontsluiten van informatieobjecten (in de 'volksmond': documenten). De component slaat deze gestructureerd en voorzien van de benodigde metadata op en stelt applicaties in staat deze te wijzigen, te verwijderen en aan de hand van een aantal zoekcriteria op te vragen. Zie ook https://www.gemmaonline.nl/index.php/GEMMA2/0.9/id-0e99ec6c-283a-4ec9-8efa-e11468e6b878 
+- EA: Enterprise Architect. Tool waarmee UML (klasse)diagrammen worden getekend.
+- EAP: Xml bestand dat als tussenbestand wordt gegenereerd uit wat is geconfigureerd in het Enterprise Architect. Bevat alle informatie die nodig is om schema's, documentatie en testberichten te genereren.
+- Extern model: TODO
+- Entiteittype: TODO
+- Gegevensmodel: In gegevensmodellen worden structuren beschreven waarmee in informatiemodellen beschreven objecten uitgewisseld worden tussen informatiesystemen. Het gaat daarbij dus NIET om complete berichtstructuren maar alleen om de content van die berichtstructuren.
+- Imvertor: Software waarmee de UML klassediagram voor een koppelvlak vertaald worden naar berichtschema's.
+- Informatiemodel: Een informatiemodel beschrijft de structuur, semantiek en eigenschappen van dingen in de werkelijkheid binnen een gegeven domein ongeacht hoe de gegevens ingewonnen, opgeslagen, beheerd en uitgewisseld worden. Het doet dat in termen van objecten, gegevens (attributen) daarvan en relaties daartussen.
+- Klassediagram: Diagram waarin entiteiten staan beschreven met hun eigenschappen, restricties en relaties.
+- Metamodel: Een metamodel is een model van een model. Een model is een abstractie van fenomenen in de echte wereld. Een metamodel is vervolgens een een abstractie van deze abstractie waarbij de eigenschappen van het model beschreven worden. Zie ook https://en.wikipedia.org/wiki/Metamodeling en https://www.noraonline.nl/wiki/Gegevensbeschrijvingen/Metagegevensmodel
+- MIG: Metamodel Informatiemodel (SIM) Gemeenten 
```

Comment:
Ik begrijp de relatie tussen MIG en SIM niet maar eigenlijk ook niet wat een MIG/MUG nu is.

Reply:
Het MIG is het model van het SIM, de beschrijving van (de structuur van) het informatiemodel.
En het MUG is het model van het UGM, de beschrijving van (de structuur van) het uitwisselingsgegevensmodel.

Eigenlijk is een MIG/MUG (metamodel) gewoon een datamodel-definitie voor een database: de beschrijving van de tabellen, indices etc.